tests/Helpers/ComponentHelpersTest.php

```diff
@@ -230,7 +218,20 @@ function ($argument) {
 	$attributes['title'] = 'Some attribute';
 
 	Components::checkAttr('bla', $attributes, $manifest, 'button');
-})->throws(\Exception::class, "bla key does not exist in the button component manifest. Please check your implementation. Check if your bla attribut exists in the component's manifest.json");
+})->throws(\Exception::class, "bla key does not exist in the button component manifest. Please check your implementation.");
+
+test('Asserts that checkAttr returns attribute based on prefix if set', function () {
+	$manifest = Components::getManifest(dirname(__FILE__, 2) . '/data/src/Blocks/components/button/');
+	$attributes = [
+		'prefix' => 'prefixedMultipleTimesButton',
+		'prefixedMultipleTimesButtonAlign' => 'right'
+	];
+
+	$results = Components::checkAttr('buttonAlign', $attributes, $manifest);
+
+	$this->assertIsString($results, 'Result should be a string');
+	$this->assertEquals('right', $results, "The set attribute should be {$attributes['prefixedMultipleTimesButtonAlign']}");
```

Comment:
```suggestion
	$this->assertSame('right', $results, "The set attribute should be {$attributes['prefixedMultipleTimesButtonAlign']}");
```

Always use strict check (type and value). `assertEquals` will just check the value.

Reply:
Got `109` results for `assertEquals` in our tests. 😄  but will change it.

Reply:
😱